src/main/java/com/dife/api/service/CommentService.java

```diff
@@ -65,14 +72,17 @@ public CommentResponseDto createComment(CommentCreateRequestDto requestDto, Stri
 		return responseDto;
 	}
 
-	public CommentResponseDto getComment(Comment comment) {
+	public CommentResponseDto getComment(Comment comment, Member member) {
 		CommentResponseDto dto = modelMapper.map(comment, CommentResponseDto.class);
 
 		dto.setLikesCount(comment.getCommentLikes().size());
 		dto.setBookmarkCount(comment.getBookmarks().size());
 
 		if (comment.getParentComment() != null) dto.setParentComment(comment.getParentComment());
 
+		if (likeCommentRepository.existsByCommentAndMember(comment, member)) dto.setIsLiked(true);
+		else dto.setIsLiked(false);
+
```

Comment:
```suggestion
dto.setIsLiked(likeCommentRepository.existsByCommentAndMember(comment, member));
```

```suggestion
boolean isLiked = likeCommentRepository.existsByCommentAndMember(comment, member);
dto.setIsLiked(isLiked);
```

`true`와 `false` 없이 위와 같이 여러 방법으로 더 깔끔하게 쓸 수도 있겠네요!

Reply:
넵!! 두번째 코드로 반영해두겠습니다! 
좋은 코드 알려주셔서 감삽니다! :)